PSKoans/Koans/Foundations/AboutLoopsAndPipelines.Koans.ps1

```diff
@@ -125,7 +125,7 @@ Describe 'Pipelines and Loops' {
                 #>
                 $i
             }
-            $Values | Should -Be @(0, 1, 2, 3, 4)
+            $Values | Should -Be __
```

Comment:
Hmm. I tend to prefer that blanks take a similar form to what's expected; we're expecting an array here, so a single blank is a bit misleading imo.

Perhaps something like this would be better?
```ps
$ExpectedValues = @(
    __
    __
    __
    __
    __
)

$ExpectedValues | Should -Be $Values
```